nextflow_schema.json

```diff
@@ -156,12 +156,16 @@
                     "exists": true,
                     "description": "Design matrix used for MAGeCK MLE to call essential genes under multiple conditions while considering sgRNA knockout efficiency"
                 },
-                "rra_contrasts": {
+                "contrasts": {
                     "type": "string",
                     "format": "file-path",
                     "exists": true,
                     "description": "Comma-separated file with the conditions to be compared. The first one will be the reference (control)"
                 },
+                "rra": {
+                    "type": "boolean",
+                    "description": "Parameter in case MAGeCK RRA should be ran instead of MAGeCK MLE."
```

Comment:
```suggestion
                    "description": "Parameter indicating if MAGeCK RRA should be ran instead of MAGeCK MLE.",
                    "default": false
```